dist/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
Do we need the .gitignore file in here?

Reply:
Yeah this just makes sure we never check in binaries.